footer: href link changed to website-goals repo

palakgupta2712 opened the pull request.

Kshashank99 reviewed (commented):
Can you please attach the before and after SS of the changes in ui .for eg. Removed code fragment in the footer.

palakgupta2712 commented:
> Can you please attach the before and after SS of the changes in ui .for eg. Removed code fragment in the footer.

The repo I cloned already had that {' '} removed.

Updated UI:
<img width="464" alt="image" src="https://user-images.githubusercontent.com/61227144/148411907-974bcb56-93ca-4c92-bac6-d4961a728a91.png">

Kshashank99 commented:
ok. Then I think those changes are not live yet.
